knimin/handlers/pm_library_prep.py

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from tornado.web import authenticated
+from tornado.escape import json_decode
+
+from knimin.handlers.base import BaseHandler
+from knimin.handlers.access_decorators import set_access
+from knimin import db
+
+
+@set_access(['Admin'])
+class PMTargetGeneLibraryPrepHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        dna_plates = db.get_dna_plate_list()
+        for plate in dna_plates:
+            plate['date'] = plate['date'].isoformat()
+
+        self.render("pm_target_gene_library_prep.html",
```

Comment:
This single renderer calls multiple similar SQL queries. Maybe there is a way to do them all at once. Just suggestion. Don't have to change now.